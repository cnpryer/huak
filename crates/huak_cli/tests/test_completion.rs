
#[cfg(test)]
mod tests {
    use std::process::Command;
    use insta_cmd::assert_cmd_snapshot;

    #[test]
    fn test_help() {
        assert_cmd_snapshot!(Command::new("huak").arg("completion").arg("--help"));
    }
}
