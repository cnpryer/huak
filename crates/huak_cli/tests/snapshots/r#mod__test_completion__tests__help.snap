---
source: crates/huak_cli/tests/test_completion.rs
info:
  program: huak
  args:
    - completion
    - "--help"
---
success: true
exit_code: 0
----- stdout -----
Generates a shell completion script for supported shells

Usage: huak completion [OPTIONS]

Options:
  -s, --shell <shell>  [possible values: bash, elvish, fish, powershell, zsh]
  -q, --quiet          
  -h, --help           Print help

----- stderr -----

